_posts/draft.md

```diff
@@ -0,0 +1,379 @@
+![Title image]({{site.url}}/assets/images/python_jobs/main.jpg)
+
+The last few months were tiring. I hardly got any time for writing. I was busy
+preparing content for a workshop and talk. Hopefully I will get into my flow
+soon. In this post I will try to answer the most important question of our
+life, “Employment!”. At conferences, IRC channels or even at the corner of a
+road, I had requested people to suggest good employment opportunities for me. I
+am not that extremely lucky fellow who has always landed on safe islands. My
+poor experiences might help you to safe-guard yourself. I am happy that at
+least I was able to knock the door of good companies. I am sharing a list of
+portals and companies that I am following or followed in last few years. As a
+bonus point, I will highlight some uncommon ways which helped me to invite new
+opportunities.  You should read this post if you are looking for **Python + web
++ (remote OR India)** related employment.
+
+*Disclaimer: I have not taken any money or any other type of bribes to advertise
+companies here.*
+
+### Strongly recommended
+
+#### [Python Job board](https://www.python.org/jobs/)
+
+This website is the best choice for finding jobs in Python. The portal is
+managed by volunteers from the Python Software Foundation. You can find both
+remote and on-site visa sponsored jobs here. You will get an option to subscribe
+via RSS feeds. They have an active Twitter handle too. I have configured
+[IFTTT](https://ifttt.com/) to drop me an email whenever new feeds are
+published.
+
+#### Mailing lists
+
+Mailing lists are good source to get notified for new job openings. In India,
+there are a few active mailing lists managed by volunteers of that community.
+Often Job emails are tagged with "[JOB]" at the subject. I have configured my
+mail client to collect all the mails with "[JOB]" in their subject at a
+dedicated folder of my mail client. This configuration helps me to pay dedicated
+attention to them. Your Hack May Vary.
+
+Below is a list of active Python communities of India.
+
+* [Bangalore](https://mail.python.org/mailman/listinfo/bangpypers)
+* [Chennai](https://mail.python.org/mailman/listinfo/chennaipy)
+* [Delhi](https://mail.python.org/mailman/listinfo/ncr-python.in)
+* [Hyderabad](https://groups.google.com/group/hyderabadpython)
+* [Kerala](https://groups.google.com/forum/#!forum/python-kerala)
+* [Pune](http://pune.python.org.in/index.html)
+
+#### [HasJobs](https://hasjob.co/)
+
+HasJob is easy to follow job portal. You will find jobs posted by emerging to
+well-established startups of India. I am following HasJobs since day one of my
+career. During my early days, I used to run a
+[bot](https://github.com/ultimatecoder/hasjobbot) to apply all the latest
+openings of the day. I am not using that tool now, but If you are interested in
+making that bot compatible with the new HasJob, please create an issue
```

Comment:
new HasJob API ??

Reply:
The tool is not based on HasJob API. It is a bot, which simulates user iteration to apply at a new opening